plugins/frontend/backstage-plugin-prometheus/src/components/PrometheusAlertStatus/PrometheusAlertStatus.tsx

```diff
@@ -149,6 +152,9 @@ export const PrometheusAlertStatus = ({
             paging: false,
             toolbar: false,
           }}
+          onRowClick={
+            onRowClick ? (_, rowData) => onRowClick?.(rowData) : undefined
```

Comment:
```suggestion
            onRowClick ? (_, rowData) => onRowClick(rowData) : undefined
```

Since you've already checked for it be to truthy, it is for sure defined.

Reply:
Resolved in latest commit